packages/vest/src/core/test/__tests__/each.test.js

```diff
@@ -0,0 +1,132 @@
+import faker from 'faker';
+import enforce from 'n4s';
```

Comment:
Better use `enforce` directly in tests so you use it from source and not from bundle.

```suggestion
import enforce from 'enforce';
```